config/routes.rb

```diff
@@ -2,9 +2,11 @@
   scope "(:locale)", locale: /en|vi/ do
     get "/login", to: "sessions#new"
     get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
     get "/about", to: "static_pages#about"
     get "/contact", to: "static_pages#contact"
     get "/service", to: "static_pages#service"
     root "static_pages#home"
+    resources :users
```

Comment:
Are you using :users for all of the restful actions? If not, than specify the actions here(for which actions you are using user resource)